### PR TITLE
Create AuthGuard Component

### PR DESCRIPTION
**Pull request by juanav22**
image

### CODE REVIEW

File: packages/next-hr-fe/src/store/utilities/authguard.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, {ReactNode} from 'react';
+import {Navigate} from 'react-router-dom';
+
+interface AuthGuardProps {
+  children: ReactNode;
```
Comment (napito27) [inline packages/next-hr-fe/src/store/utilities/authguard.tsx]: `React.ReactNone`  si lo usas asi no tienes que importar ReactNode

File: packages/next-hr-fe/src/routing/app.tsx [hunk 2]
```diff
@@ -16,6 +17,23 @@ function App() {
...
+            element={
+              <AuthGuard>
+                <PlayGround />
+              </AuthGuard>
+            }
+          />
+          <Route
+            path="/api-playground"
+            element={
+              <AuthGuard>
```
Comment (napito27) [inline packages/next-hr-fe/src/routing/app.tsx]: Is there a way to do it without having to repeat this code
`<AuthGuard>`?

**Review by napito27 — COMMENTED** [review]
left some comments

**Review by napito27 — APPROVED** [review]
Great job Ingeniera

**Review by paulagutierrez08 — APPROVED** [review]
Good Job engineer!

**Pull request merged**